controllers/logs.js

```diff
@@ -0,0 +1,75 @@
+const express = require('express');
+const router = express.Router();
+
+const url = require('url');
+const crypto = require('crypto');
+const ipaddr = require('ipaddr.js');
+
+const api = require('../api');
+const helper = require('../helpers/googleAnalytics');
+
+const winston = require('winston');
+const logger = winston.createLogger({
+    transports: [
+        new winston.transports.Console({
+            format: winston.format.combine(
+                winston.format.colorize(),
+                winston.format.simple()
+            )
+        })
+    ]
+});
+
+logger.info('Google Analytics Tracking ID: ' + process.env.SW_GOOGLE_ANALYTICS_TRACKING_ID);
+
+router.use(require('../helpers/authentication').authChecker);
+
+function idCleanup(url){
+    const match = /\/[0-9a-f]{24}/g;
```

Comment:
please add comment that this might result in false postives if url slugs are exactly 24 chars long